### PR TITLE
Document private extension guideline in styleguide

### PR DESCRIPTION
**Pull request by dgalarza**

### CODE REVIEW

**Comment by sidraval** [comment]
I'm into this :+1:

File: style/swift/README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ Swift
 * Let the compiler infer the type whenever possible
 * Group computed properties below stored properties
 * Use a blank line above and below computed properties
+* Group `private` methods in a `private` `extension`
```
Comment (gfontenot) [inline style/swift/README.md]: how about

> group methods with different levels of access control in specific extensions.

That's horribly worded, but the gist is: we should be doing this for `public` as well.

Comment (gilesvangruisen) [inline style/swift/README.md]: maybe:
> group methods into specific extensions for each level of access control

**Comment by dgalarza** [comment]
Updated to make this guideline not so specific to private extensions, but to grouping by level of access control.

**Comment by gfontenot** [comment]
Dig it :+1:

**Pull request merged**